platform/ios/src/MGLMapboxEvents.h

```diff
@@ -4,29 +4,20 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/// Event types
```

Comment:
`///` creates a documentation comment. You probably didn’t intend to document `MGLEventTypeAppUserTurnstile` individually with this comment.